src/edu/washington/escience/myria/operator/SymmetricHashJoin.java

```diff
@@ -331,7 +331,13 @@ public SymmetricHashJoin(final List<String> outputColumns, final Operator left,
   @Override
   protected Schema generateSchema() {
     final Schema leftSchema = getLeft().getSchema();
+    if (leftSchema == null) {
```

Comment:
What's the new case being handled here? Is this the only operator that needs to handle this case?